app/shared/components/Global/Form/Field/Key/Public.js

```diff
@@ -6,25 +6,37 @@ import { Form, Input } from 'semantic-ui-react';
 import debounce from 'lodash/debounce';
 
 export default class GlobalFormFieldKeyPublic extends Component<Props> {
-  state = { value: '' };
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: props.defaultValue
+    };
+  }
+
   onChange = debounce((e, { name, value }) => {
     const parsed = value.trim();
+    const valid = !!(parsed.match(/^EOS[a-zA-Z1-9.]{50}?$/g));
+
     this.setState({
       value: parsed
     }, () => {
-      this.props.onChange(e, { name, value: parsed });
+      this.props.onChange(e, { name, value: parsed, valid });
     });
-  }, 300)
+  }, 1000)
```

Comment:
Try reducing this back down to 300 with `the ecc.isValidPublic` call.

Reply:
I actually meant to change that back and forgot lol, thanks for pointing it out.